astropy/io/fits/connect.py

```diff
@@ -435,7 +435,7 @@ def write_table_fits(input, output, overwrite=False, append=False):
     ----------
     input : Table
         The table to write out.
-    output : str
+    output : str or os.PathLike[str]
```

Comment:
not sure what the `[str]` suffix means in this case ?

Reply:
perhaps a simpler way to go would be
```suggestion
    output : str or pathlib.Path
```

Reply:
I probably authored most of these annotations, so it is fine as is, AFAIC, I'm just offering an alternative in case @saimn has a different preference